src/apiCalls.tsx

```diff
@@ -42,3 +44,43 @@ export function getCreatorInfo(id: number) {
       throw error; 
     });
 }
+
+export function postCreator(creator: CreatorPost) {
+  return fetch('https://be2310-social-media-aggregator-18a4f3a92617.herokuapp.com/api/v1/users/1/follows', {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify(creator)
+  })
+    .then(response => {
+      if (response.ok) {
+        return response.json();
+      } else {
+        throw new Error(`Failed to post creator`);
+      }
+    })
+    .catch(error => {
+      console.error('Error posting creator:', error);
+      throw error; 
+    });
+}
+
+export function deleteCreator(id: number) {
+  return fetch(`https://be2310-social-media-aggregator-18a4f3a92617.herokuapp.com/api/v1/users/1/follows/${id}`, {
+    method: 'DELETE',
+    headers: { 'Content-Type': 'application/json' }
+  })
+    .then(response => {
```

Comment:
Nice job adding error handling on the deleteCreator fetch, good to let the user know if they successfully unfavourite a creator